unyt/dimensions.py

```diff
@@ -27,12 +27,16 @@
 angle = Symbol("(angle)", positive=True)
 #: current_mks
 current_mks = Symbol("(current_mks)", positive=True)
+#: amount
+amount = Symbol("(amount)", positive=True)
```

Comment:
I'm not sure how I feel about needing to define this. There are certainly situations where it's convenient to think of a mol as just a number. I don't do chemistry calculations often enough to have a strong opinion though.